packages/affiliate/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "automattic/jetpack-affiliate",
+	"description": "Reads an affiliate code for Jetpack. Useful to add it to the connection URL so it's read by wpcom.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.0.0",
```

Comment:
Could we omit the version? Ideally, once this is published as a proper package, its corresponding GH repo release tags should be the canonical source of truth.

Reply:
Done.